plugins/uefi-dbx/README.md

```diff
@@ -74,4 +73,5 @@ This plugin requires:
 
 ## Version Considerations
 
-This plugin has been available since fwupd version `1.5.0`.
+This plugin has been available since fwupd version `1.5.0` with `org.uefi.dbx-isodate`
+being available from fwupd version 2.0.4.
```

Comment:
1.9.27 in 1.9.x  series and 2.0.4 in 2.0.x series.

Reply:
Ack, fixed.